ajax/libs/responsive-carousel/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "responsive-carousel",
+  "filename": "responsive-carousel.min.js",
+  "description": "A jQuery-based script for responsive carousels that work with mouse, touch, and keyboard",
+  "homepage": "https://master-origin-responsive-carousel.fgview.com/test/functional/flip-auto.html",
+  "author": {
+    "name": "Filament Group, Inc.",
+    "url": "https://filamentgroup.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/filamentgroup/responsive-carousel"
+  },
+  "keywords": [
+    "responsive",
+    "carousel"
+  ],
+  "autoupdate": {
+    "source": "git",
```

Comment:
`src` is available in the NPM package linked. I would prefer to use the NPM package as the source for the auto-update where possible, as files contained are normally more consistent that what people keep in a repo and decide to tag.